assets/collage.css

```diff
@@ -29,11 +33,7 @@
 
 @media screen and (min-width: 750px) {
   .collage {
-    grid-template-columns: 2fr 1fr;
-  }
-
-  .collage--right {
-    grid-template-columns: 1fr 2fr;
+    grid-template-columns: 1fr 1fr 1fr;
```

Comment:
Based on what Olivia noticed, it means that we would need this to be `repeat(6, 1fr)` I think. So it's a multiple of `3` and `2`. And then the rest of the `grid-column` allocation needs to be updated 🤔

Reply:
I just made the changes and tested all scenarios and it works 🤔 :

https://screenshot.click/13-03-15lza-acv3i.mp4

let me know if I missed anything